eslint/backend.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+  extends: [
+    "./index.js",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:prettier/recommended" // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+  ],
+  plugins: [
+    "typescript-eslint",
+  ],
+  parser: "@typescript-eslint/parser",
+  ignorePatterns: ["node_modules/*", "built/*", "lib/*", "build/*"],
```

Comment:
We should standardize our tsconfig outDir folder for all repositories. Right now, we have: lib, build, and built depending on repos. I would go for built 😊

Reply:
built sounds good to me!